com.unity.shadergraph/Documentation~/Swizzle-Node.md

```diff
@@ -17,15 +18,12 @@ Channel dropdown parameters are dynamic depending on the length of the input vec
 
 | Name        | Type           | Options  | Description |
 |:------------ |:-------------|:-----|:---|
-| Red out      | Dropdown | Red, Green, Blue, Alpha (depending on input vector dimension) | Defines which input channel should be used in the output's red channel |
-| Green out      | Dropdown | Red, Green, Blue, Alpha (depending on input vector dimension) | Defines which input channel should be used in the output's green channel |
-| Blue out      | Dropdown | Red, Green, Blue, Alpha (depending on input vector dimension) | Defines which input channel should be used in the output's blue channel |
-| Alpha out      | Dropdown | Red, Green, Blue, Alpha (depending on input vector dimension) | Defines which input channel should be used in the output's alpha channel |
+| Mask     | Inputfield | X, Y, Z, W (depending on input vector dimension) | The swizzle mask is a combination of one to four characters that can be x, y, z, or w. The size of output value depends on the length of the mask input.|
```

Comment:
I'm also able to use r, g, b, a as mask inputs. So would add a mention of that here.